openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -86,7 +97,7 @@ function combineIndicators(nameLookUp, themeObj, JSONthemes) {
             // compute the themes
             for (var w = 0; w < themeKeys.length; w++) {
                 var tempThemeName = themeKeys[w];
-                tempElementValue = tempElementValue + themeObj[v][tempThemeName];
+                tempElementValue = (tempElementValue + themeObj[v][tempThemeName]) * inversionFactor;
```

Comment:
I am not sure it will work correctly. Please double-check if you are actually considering for each node it's own inversion factor, and multiplying the node's value by it